app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -701,18 +701,30 @@ export default {
         if (this.isSignatureEnabledForInbox && this.messageSignature) {
           newMessage += '\n\n' + this.messageSignature;
         }
-        const messagePayload = this.getMessagePayload(newMessage);
+
+        if (this.isWhatsAppChannel) {
+          this.sendMessageAsMultipleMessages(newMessage);
+        } else {
+          const messagePayload = this.getMessagePayload(newMessage);
+          this.sendMessage(messagePayload);
+        }
 
         this.clearMessage();
```

Comment:
Line 717 has another `clearMessage` call, we can probably get rid of one

Reply:
I have removed the extra clearMessage call.